hosting_docs/source/reference/1-commcare-cloud/1-installation.rst

```diff
@@ -152,23 +152,42 @@ Manual setup
 If you'd rather use your own virtualenv name or a different commcare-cloud repo
 location, or if the script above did not work.
 
+Setup and activate the virtualenv
+"""""""""""""""""""""""""""""""""
+
+**NOTE**: *The virtualenv name and location may be customized, below example uses ``cchq``
+and ``~/.virtualenvs/cchq``. Adjust according to your preferred configuration.*
+
+.. code-block:: sh
+
+   # using virtualenv
+   python3 -m pip install --user --upgrade virtualenv
+   python3 -m virtualenv ~/.virtualenvs/cchq
+   source ~/.virtualenvs/cchq/bin/activate
```

Comment:
a) I think we need to be explicit about python3.10 because this is outside of a virtual environment, and we don't know for sure what version python3 references. 
b) We are shifting to removing the dependency on `virtualenv` since python has `venv` built-in. 
```suggestion
   # using venv
   python3.10 -m venv ~/.virtualenvs/cchq
```